combat-trainer.lic

```diff
@@ -2664,6 +2664,10 @@ class AttackProcess
       game_state.action_taken
       ammo = Regexp.last_match(2)
       bput("stow my #{ammo}", 'You pick up', 'Stow what')
+      if game_state.dual_load?
+        ammo = Regexp.last_match(2)
```

Comment:
💡 I think `ammo` here will be the same as the previous assignment, so this could be removed?